src/PrestaShopBundle/Form/Admin/Sell/Customer/CustomerType.php

```diff
@@ -107,6 +107,17 @@ public function __construct(
      */
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
+        if ($options['show_guest_field'] === true) {
+            $builder
+                ->add('is_guest', SwitchType::class, [
+                    'label' => $this->trans('Register as guest', 'Admin.Global'),
```

Comment:
Hi :) I think we should avoid having the word "register" and "guest" in the same sentence. In my opinion, It's kind of confusing. We could use "Guest checkout" as a label. I think this expression is quite known in ecommerce, so I'm not sure it requires a help text. What could be nice would be to display this feature just under the password field and have the password field grayed out when this feature is enabled? What do you think?

Reply:
Hello Julie, I am not against some other label, but I don't like `Guest checkout`, because we are not in a 'checkout' when creating the customer in backoffice. Can you try to come up with some other options? 😉

Reply:
Yes, sorry, I read too quickly and I thought we were on the back-office order page.  What about "Guest" only? And for the help text:

'Continue without creating a customer account"

Reply:
What about "Create a guest"?

`Continue without creating a customer account` - Well, you create a customer account, but of a guest type. I think the help text should mean something like: "This will create a guest account with no password and a default guest group. Customer won't be able to log in with this account, unless it's converted to normal customer account. This creates the same type of customer as if customers does guest checkout in the front office." ➡️ Can you make a help text out of this? 😄

Reply:
Ok, well, I think the misunderstanding comes from the fact that, in my opinion, a guest is not a customer. And I wouldn't talk about a "guest account". It's confusing, as being a guests means that you don't have an account in the FO.

But I understand your point. 

Here's another suggestion:

The user won't have a password to sign in to your store unless you turn their account into a customer one.

(The merchant will have to do the modification, right?)

Reply:
- They can convert their guest account on order confirmation or guest tracking page.
- They are customers. The only difference is is_guest flag being true + not being able to log in with it. They are in the same table, in the same listing, the same entity. :-)

Reply:
We can just call it:
`Create guest account`
`This will create a single use account with no password. User won't be able to sign in with it, unless he converts it to a regular account.`

Reply:
Ok, thanks for your explanation. The following suggestion should do it then. :) 
Label: "Create a guest" as you suggested
Help text: "The user won't have a password to sign in to your store unless they create an account."

Reply:
@Julievrz They don't create an account, they convert it from guest to customer type.